src/main/java/seedu/lifeasier/tasks/Deadline.java

```diff
@@ -5,6 +5,8 @@
 public class Deadline extends Task {
     protected LocalDateTime by;
 
+    private static final String TYPE = "deadline";
```

Comment:
Consider changing these to non-static variables. Such that we would be accessing the object for the getType() method instead of accessing the class.